exercises/bowling/example.rb

```diff
@@ -20,7 +20,7 @@ def roll(pins)
   end
 
   def validate(pins)
-    raise 'Invalid number of pins' unless (RULES[:MIN]..RULES[:MAX]).cover?(pins)
+    raise 'Invalid number of pins' unless (PINS[:MIN]..PINS[:MAX]).cover?(pins)
```

Comment:
So I think for this aspect, the discussion should go to x-common... find out if this is an aspect for others, and if we can change it in common.